### DISCUSSION TITLE
Unable to see tracking on message activity option

### DISCUSSION THREAD

**Discussion by contactosama** [postalserver/postal#2553] [Help with using Postal]
![Screenshot_20230720_224506_Chrome.jpg](https://github.com/postalserver/postal/assets/105778300/e0ac9829-e527-486e-9068-6967ffdb6406)

I have added track domain, but unable to see 'click for '

Like the below picture, please help

![tracked-message.png](https://github.com/postalserver/postal/assets/105778300/4e37f6f9-9d6a-4ba3-995a-f0e5489180fa)

**Comment by willpower232** [comment]
Have you enabled click and open tracking for this server?

**Reply by contactosama** [reply]
Can you please share how to enable it?
I had configured caddy file and domain cname mapping

**Reply by willpower232** [reply]
Have you added a tracking domain in the postal UI as per the rest of the page you got the screenshot from?

https://docs.postalserver.io/features/click-and-open-tracking

**Reply by contactosama** [reply]
![image](https://github.com/postalserver/postal/assets/105778300/17119d28-c839-4b97-96fc-db812291ca6b)

**Reply by contactosama** [reply]
I sent many test emails from Postal UI but no tracking activity showing

**Reply by willpower232** [reply]
it says the cname is configured correctly so what you need to do is visit the tracking domain and see if you see the word OK in case your server configuration isn't working or the DNS isn't resolving

**Reply by contactosama** [reply]
![image](https://github.com/postalserver/postal/assets/105778300/5b74dd8c-5470-4ab9-93cf-bd28ac4f3029)

**Reply by wmr-trevor** [reply]
Don't know if you got this resolved, but I'd suggest looking at #2415 and see if downgrading to 2.1.2 works for you.
